Google.PowerShell/ReleaseFiles/GoogleCloud.psm1

```diff
@@ -1,4 +1,6 @@
-﻿$script:GCloudModule = $ExecutionContext.SessionState.Module
+﻿param()
```

Comment:
Should we define an argument list in the param block? Something like [this](https://stackoverflow.com/questions/36897511/powershell-module-pass-a-parameter-while-importing-module)

Reply:
Do we really want any additional parameters to be ordered? If we do, that is fine, but if we don't, what I have written is orderless.